RC: Update Azure AD SAML Guide to Microsoft Entra ID and add Namespace note (#325)
redis/docs: content/operate/rc/security/access-control/saml-sso/saml-integration-azure-ad.md

---

> Title: Microsoft Entra SAML integration guide

Author: ```suggestion
Title: Microsoft Entra (formerly Azure Active Directory) SAML integration guide
```

alwaysopen: false
categories:
- docs
- operate
- rc
description: This integration guide shows how to set up Microsoft Entra as a SAML single sign on provider for your Redis Cloud account.
linkTitle: Microsoft Entra integration
weight: 10
---

> This guide shows how to configure [Microsoft Entra](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-architecture) as a SAML single sign-on identity provider (IdP) for your Redis Cloud account.

Author: ```suggestion
This guide shows how to configure [Microsoft Entra](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-architecture) (formerly Azure Active Directory) as a SAML single sign-on identity provider (IdP) for your Redis Cloud account.
```

    {{<image filename="images/rc/saml/sm_saml_8.png" >}}

> 1. A popup appears, explaining that to test the SAML connection, you need to log in with the credentials of a user defined in Microsoft Entra.

Reviewer: Minor issue, but "that to test" reads a bit awkwardly for me.
```suggestion
1. A popup appears, explaining that you must log in with the credentials of a Microsoft Entra user to test the SAML connection.
```
